ballerina/client.bal

```diff
@@ -16,11 +16,11 @@
 
 import ballerina/jballerina.java;
 
-# Represents an SQL client.
-#
+# Represents an SQL client. It should be kept open throughout the entirety of the application to perform the operations.
```

Comment:
```suggestion
# Represents an SQL client.
```
Lets not add it here, it should be added to the constructor. Lets add it to other database connectors.